libraries/chain/database.cpp

```diff
@@ -2287,9 +2349,17 @@ namespace golos { namespace chain {
 
                     unclaimed_rewards = 0;
                 }
-                else if (has_hardfork(STEEMIT_HARDFORK_0_19__898) && c.total_vote_weight > 0) {
+                // Case: auction window destination is reward fund
+                else if (has_hardfork(STEEMIT_HARDFORK_0_19__898) &&
+                        c.total_vote_weight > 0 && c.auction_window_reward_destination == protocol::to_reward_fund
+                        || // Also in case when there are no votes after end of auction window,
+                        votes_after_auction_window_weight == 0 // tokens must go to reward fund
+                    ) {
```

Comment:
Here should be checking on `c.auction_window_reward_destination != protocol::to_author`.
If any unclaimed rewards exist, they should be returned to reward fund after HF19